Makefile

```diff
@@ -58,9 +58,9 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	pip-compile --upgrade -o requirements/quality.txt requirements/quality.in
 	pip-compile --upgrade -o requirements/travis.txt requirements/travis.in
 	pip-compile --upgrade -o requirements/dev.txt requirements/dev.in
-	# Let tox control the Django version for tests
-	sed '/^[d|D]jango==/d' requirements/test.txt > requirements/test.tmp
-	mv requirements/test.tmp requirements/test.txt
+	# Let tox control the Django and djangorestframework versions for tests
+	sed -i"" '/^[d|D]jango==/d' requirements/test.txt
+	sed -i"" '/^djangorestframework==/d' requirements/test.txt
```

Comment:
Need a space, or this is the same as no argument.

Reply:
Updated to work on both BSD and linux.